aten/src/ATen/native/ChanelShuffle.cpp

```diff
@@ -28,6 +28,7 @@ Tensor channel_shuffle(const Tensor& self, int64_t groups) {
               "channel_shuffle expects input with > 2 dims, but got input with sizes ",
               self.sizes());
   int64_t c = self.size(1);
+  TORCH_CHECK(self.numel() != 0, "nn.ChannelShuffle: Input tensor must not be empty");
```

Comment:
Alternatively, we could just return the `self` tensor for this case. This fits with the semantics of [`nn.ChannelShuffle`](https://pytorch.org/docs/stable/generated/torch.nn.ChannelShuffle.html) and I think it's better to not error out if we can help it.